### DISCUSSION TITLE
Can I disable KHR_texture_transform plugin?

### DISCUSSION THREAD

**Discussion by zhbtman123** [zeux/meshoptimizer#327] [Q&A] [answered]
I'm use meshopt now 

And I found meshopt use 'KHR_texture_transform' when gltf has texture . generate rotation  translate etc...  

but when I set a mesh which without texture( has UV ) to meshopt . get optimizer gltf & bin  and load in my project(use threejs) set texturre . the view is incorect ( like has no UV,but I fonud UV date on attribute.Uv)

**Comment by zeux** [comment, marked as answer]
You need to either use `-noq` or add a dummy texture to your scene and replace it at runtime, see #280.
